saleor/graphql/attribute/utils/shared.py

```diff
@@ -27,6 +28,12 @@
 )
 
 
+@dataclass
+class AssignedAttributeData:
+    attribute: ChannelContext[attribute_models.Attribute]
```

Comment:
Should we use a name like `attribute_with_context` to increase code readability?
When in the resolver, I see `channel_slug = root.attribute.channel_slug` I could think about attribuite as a model.

Reply:
added here: https://linear.app/saleor/issue/ENG-657/value-of-limit-for-assignedattribute-types-should-be-used-to-limit-the